### ISSUE TITLE
vectorization of words

### ISSUE THREAD

**Issue by ghost** [tobigithub/tensorflow-deep-learning#16]
Hello, 
I installed tensorflow on windows and executed this tutorial as python word2vec.py and got the following errors

![word2vec](https://cloud.githubusercontent.com/assets/20130992/24427960/d15aa570-13da-11e7-98d9-ba86568e1da1.PNG)

**Comment by tobigithub** [comment]
Hi,
Windows can be tricky. Best is to follow step by step the installation from
https://www.tensorflow.org/install/install_windows

I am quite sure you need **wget** for windows, so you need to download it from here https://www.gnu.org/software/wget/

or better install all the Unix tools for Windows from 
https://www.cygwin.com/

**Comment by ghost** [comment]
Yes I deleted and installed tensorflow in Ubuntu. I am not able to proceed with vectorization of words. I have the word2vec.py. i do not know how to execute this tutorial. Can anyone please tell me the execution steps for this tutorial
